src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -43,6 +43,8 @@ public sealed class LanguageService
 
         internal const int OnConnectionWaitTimeout = 300000;
 
+        internal const int PeekDefinitionTimeout = 10000;
```

Comment:
>10000 [](start = 51, length = 5)

Minor: Is this 10 seconds? One thing we do in other projects is to define a constant for 1 second and use 10 * Seconds to do this kind of definition. It's handy as it makes clear the unit of measurement. Alternatively a comment also keep this clear.